lxd/storage/drivers/driver_btrfs_utils.go

```diff
@@ -95,7 +96,7 @@ func (d *btrfs) getSubvolumes(path string) ([]string, error) {
 	}
 
 	// Walk through the entire tree looking for subvolumes.
-	err := filepath.Walk(path, func(fpath string, fi os.FileInfo, err error) error {
+	err := filepath.WalkDir(path, func(fpath string, entry fs.DirEntry, err error) error {
```

Comment:
And this is quicker too!

Reply:
It was easy because the docstring kind of contained the solution :)